frame/collective/src/lib.rs

```diff
@@ -644,11 +643,16 @@ fn get_result_weight(result: DispatchResultWithPostInfo) -> Option<Weight> {
 }
 
 impl<T: Config<I>, I: 'static> Pallet<T, I> {
+	/// Actual proposal for a given hash, if it's current.
+	pub fn proposal_of(who: T::Hash) -> Option<<T as Config<I>>::Proposal> {
+		ProposalOf::<T, I>::get(who)
+	}
+
```

Comment:
do we need this here? I can't find any usage of the verbosely defined getter functions in the rest of the code